examples/src/main/scala/org/apache/spark/examples/SparkKMeans.scala

```diff
@@ -75,7 +74,10 @@ object SparkKMeans {
     val data = lines.map(parseVector _).cache()
     val K = args(1).toInt
     val convergeDist = args(2).toDouble
-
+    /*
+     * A requirement of using takeSample() method: the element in data (line 74) should be unique (No duplicates), otherwise the size of newPoints (line 89) will not 
```

Comment:
This sounds like a comment for a caller, but it's not visible to anyone but those reading source code. 

Why not just call `.distinct` in the code to avoid the problem entirely?